pkg/speaker/vip/keepalived.go

```diff
@@ -106,6 +105,22 @@ func (k *KeepAlived) Start(stopCh <-chan struct{}) error {
 	return err
 }
 
+// User can config Keepalived by ConfigMap to specify the images
+// If the ConfigMap exists and the configuration is set, use it,
+// 	otherwise, use the default image got from constants.
+func (k *KeepAlived) getConfig() (*corev1.ConfigMap, error) {
+	return k.clientset.CoreV1().ConfigMaps(util.EnvNamespace()).
+		Get(context.Background(), constant.OpenELBImagesConfigMap, metav1.GetOptions{})
+}
+
+func (k *KeepAlived) getImage() string {
+	cm, err := k.getConfig()
+	if err != nil {
+		return constant.OpenELBDefaultKeepAliveImage
+	}
+	return cm.Data[constant.OpenELBKeepAliveImage]
```

Comment:
`cm.Data[constant.OpenELBKeepAliveImage]`
Is it possible for null pointer to occur here?

Reply:
oh, may be empty